delius-stage/sub-projects/new-tech.tfvars

```diff
@@ -30,23 +30,38 @@ pdfgenerator_conf = {
 # Override default Offender API Config
 offenderapi_conf = {
   env_oracledb_servicename = "STGNDA_TAF"
-  image_version = "0.1.20"
+  image_version            = "0.1.20"
```

Comment:
I removed the env specific versions so it just gets whatever is live

Reply:
Have removed as suggested.